stores/preferences.ts

```diff
@@ -29,6 +29,7 @@ interface State {
   enableAllArtwork: boolean
   enableGyroEffect: boolean
   gridSize: 'small' | 'medium' | 'large'
+  subscribedToNewsletter: boolean
```

Comment:
This will be reset when logging out or otherwise clearing browser storage

But we don't have any permanent storage of user info, do we?
@kodadot/internal-dev

Reply:
Nope, try localstorage or cookies

Reply:
good q. or maybe time to create one (new service or store it on chain)?

Reply:
> new service 

maybe worker? or basic supabase

> or store it on chain

will cost $ each tx and I don't think that's the point